ui/assets/css/_header.scss

```diff
@@ -22,3 +21,14 @@
     padding-left: 0.2rem;
   }
 }
+
+@media (max-width: 52em) {
```

Comment:
@cmc333333 I didn't do that because the basscss variables house "min-width: 52em".